deck-builder/main.js

```diff
@@ -1272,10 +251,134 @@ $cardList.addEventListener('click', function (event) {
       class: event.target.closest('tr').className
     }
   }
-  addCard($card)
+  var fullCardInfo = addCard($card)
   var deckCount = deckCounter()
   $characterCounter.textContent = deckCount.characterCount
   $eventCounter.textContent = deckCount.eventCount
   $climaxCounter.textContent = deckCount.climaxCount
   $cardCounter.textContent = deckCount.cardCount + '/50'
+
+  if (fullCardInfo !== null) {
+    if (uniqueCardNumbers.hasOwnProperty(fullCardInfo.id)) {
+      var $selectElementsList = document.querySelectorAll('select')
+      for (var i = 0; i < $selectElementsList.length; i++) {
+        if ($selectElementsList[i].hasAttribute('card-number')) {
+          if ($selectElementsList[i].getAttribute('card-number') === fullCardInfo.id) {
+            var $targetSelectElement = $selectElementsList[i]
+          }
+        }
+      }
+
+      if (uniqueCardNumbers[fullCardInfo.id] === 1) {
+        $targetSelectElement.options[1].removeAttribute('selected')
+        $targetSelectElement.options[2].setAttribute('selected', '')
+      }
+      else if (uniqueCardNumbers[fullCardInfo.id] === 2) {
+        $targetSelectElement.options[2].removeAttribute('selected')
+        $targetSelectElement.options[3].setAttribute('selected', '')
+      }
+      else if (uniqueCardNumbers[fullCardInfo.id] === 3) {
+        $targetSelectElement.options[3].removeAttribute('selected')
+        $targetSelectElement.options[4].setAttribute('selected', '')
+      }
+      uniqueCardNumbers[fullCardInfo.id] = uniqueCardNumbers[fullCardInfo.id] + 1
+    }
+    else {
+      uniqueCardNumbers[fullCardInfo.id] = 1
+      if (fullCardInfo.cardType === 'Character') {
+        if (fullCardInfo.level === 0) {
```

Comment:
Try a lookup table instead of a large set of `else if`s.

```js
var levels = [$level0, $level1, $level2, $level3]
if (fullCardInfo.cardType === 'Character') {
  levels[fullCardInfo.level].appendChild(renderDeckListCard(fullCardInfo))
}
```

Where else can you make your code more concise?